### PR TITLE
Update troubleshooting and kubeadm labs

### PR DESCRIPTION
**Pull request by victorvarza**
Update the following labs:
- troubleshooting
  - refactor manifests
   - remove unnecessary files
- kubeadm
  - update ubuntu version to focal
  - update flannel v0.15.1

### CODE REVIEW

File: src/networking/l7-example.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 apiVersion: "cilium.io/v2"
 kind: CiliumNetworkPolicy
+description: "Allow HTTP GET /public from env=prod to app=service"
```
Comment (adriananeci) [inline src/networking/l7-example.yaml:3]: This won't work in cilium 1.11 anymore. I think you forgot to rebase from master. See https://github.com/hlesey/k8s-labs/pull/17

File: src/storage/pv-local.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 kind: PersistentVolume
 apiVersion: v1
 metadata:
-   name: local001
+   name: local01
```
Comment (adriananeci) [inline src/storage/pv-local.yaml:4]: Please rebase from master. See https://github.com/hlesey/k8s-labs/pull/17

File: src/storage/pvc-local.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-   name: pvc000
+   name: pvc00
```
Comment (adriananeci) [inline src/storage/pvc-local.yaml:4]: Please rebase from master

**Review by adriananeci — CHANGES_REQUESTED** [review]
Small nit

**Review by adriananeci — APPROVED** [review]

File: src/networking/l7-example.yaml (hunk 1 above)
Comment (victorvarza) [reply src/networking/l7-example.yaml]: reverted

File: src/storage/pvc-local.yaml (hunk 3 above)
Comment (victorvarza) [reply src/storage/pvc-local.yaml]: reverted

File: src/storage/pv-local.yaml (hunk 2 above)
Comment (victorvarza) [reply src/storage/pv-local.yaml]: reverted

**Pull request merged**